examples/sync-pass/src/sync-pass.rs

```diff
@@ -89,24 +89,13 @@ fn update_string(field_name: &str, field: &mut String, extra: &str) -> bool {
     }
 }
 
-fn update_encrypted_string(
-    field_name: &str,
-    field: &mut String,
-    extra: &str,
-    encdec: &EncryptorDecryptor,
-) -> bool {
-    let opt_s = prompt_string(format!(
-        "new {} [now {}{}]",
-        field_name,
-        encdec.decrypt(field).unwrap(),
-        extra
-    ));
-    if let Some(s) = opt_s {
-        *field = encdec.encrypt(&s).unwrap();
-        true
-    } else {
-        false
-    }
+fn update_encrypted_fields(fields: &mut EncryptedFields, extra: &str) {
```

Comment:
nit: Seeing `EncryptedFields` outside of the `Login` struct makes me want it to be renamed `EncryptedLoginFields` so this struct isn't construed as a generic struct not associated with logins. Although perhaps it would read as redundant nested inside of the `Login` struct but maybe worth it? Do what you will with this.